src/component/Navbar/Navbar.scss

```diff
@@ -0,0 +1,78 @@
+.Navbar {
+  background-color: #644d46;
```

Comment:
do you want to use here global variable --color-liver: #644D46 ?

Reply:
changed